### PR TITLE
projects checklist

### PR DESCRIPTION
**Pull request by sofer**
@iteles anything to add?

### CODE REVIEW

File: projects/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
++ Issues
+  + How many are there?
+  + Any interesting ones?
+  + Have they been assigned?
+  + Have they been labelled?
+  + Have any issues been closed by anyone other than the person that raised them?
++ Commit history
+  + Do the commits tell a story?
++ Tests
++ Site
```
Comment (iteles) [inline projects/README.md:26]: Does this mean 'does the site exist or do I have to run this locally?' or 'You should have deployed to Heroku and included the link in your readme?'

File: projects/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
++ Code
```
Comment (iteles) [inline projects/README.md:27]: What does this mean? That there should be some code? :wink: I'm not sure who your target audience for this is! The students? The end of week reviewer?

**Comment by iteles** [comment]
Ooooooooooh, is this a list of things that we should be looking at when FAC6 do their end of week project presentations?

**Comment by sofer** [comment]
@iteles exactly! And it's not exactly complete. 

On reflection, I am not sure why I have made it the README. It might be better named as _endofweek.md_

**Comment by iteles** [comment]
@sofer It would make sense as _part_ of the readme, you just need to add a short introductory paragraph what the contents of the folder are and what the context is for the checklist :blush:

**Comment by sofer** [comment]
@iteles done.

**Pull request merged**